plexpy/notification_handler.py

```diff
@@ -43,7 +45,8 @@ def notify(stream_data=None, notify_action=None):
                         # Set the notification state in the db
                         set_notify_state(session=stream_data, state='play', agent_info=agent)
 
-                    elif agent['on_stop'] and notify_action == 'stop':
+                    elif agent['on_stop'] and notify_action == 'stop' \
+                        and (plexpy.CONFIG.NOTIFY_CONSECUTIVE or progress_percent < plexpy.CONFIG.NOTIFY_WATCHED_PERCENT):
```

Comment:
Shouldn't this be `plexpy.CONFIG.NOTIFY_CONSECUTIVE and progress_percent < plexpy.CONFIG.NOTIFY_WATCHED_PERCENT`?